2008-02-14  H.J. Lu  <hongjiu.lu@intel.com>

	* ld-shared/sh1.c (shlib_overriddencall2): Moved to ...
	* ld-shared/sh2.c (shlib_overriddencall2): Here.  New.

--- ld/testsuite/ld-shared/sh1.c	9 Jun 1999 03:04:03 -0000	1.2
+++ ld/testsuite/ld-shared/sh1.c	14 Feb 2008 20:23:27 -0000	1.3
@@ -71,12 +71,6 @@ shlib_shlibcall2 ()
 {
   return shlib_overriddencall2 ();
 }
-
-int
-shlib_overriddencall2 ()
-{
-  return 7;
-}
 #endif
 
 /* This function calls a function defined by the main program.  */
--- ld/testsuite/ld-shared/sh2.c	3 May 1999 07:29:09 -0000	1.1.1.1
+++ ld/testsuite/ld-shared/sh2.c	14 Feb 2008 20:23:27 -0000	1.2
@@ -12,3 +12,11 @@ shlib_shlibcalled ()
 {
   return 5;
 }
+
+#ifndef XCOFF_TEST
+int
+shlib_overriddencall2 ()
+{
+  return 7;
+}
+#endif
